src/uct/ib/base/ib_iface.c

```diff
@@ -1084,9 +1084,7 @@ static void uct_ib_iface_set_num_paths(uct_ib_iface_t *iface,
     if (config->num_paths == UCS_ULUNITS_AUTO) {
         if (uct_ib_iface_is_roce(iface)) {
             /* RoCE - number of paths is RoCE LAG level */
-            iface->num_paths =
-                    uct_ib_device_get_roce_lag_level(dev, iface->config.port_num,
-                                                     iface->gid_info.gid_index);
+            iface->num_paths = dev->lag_level;
```

Comment:
still need the fallback to sysfs for device which does not support devx

Reply:
Fixed.